tools/SourceKit/tools/sourcekitd/bin/InProc/sourcekitdInProc.cpp

```diff
@@ -111,14 +111,24 @@ void sourcekitd::set_interrupted_connection_handler(
 // sourcekitd_request_sync
 //===----------------------------------------------------------------------===//
 
+/// Create a new SourceKit request handle. Each call of this method is
+/// guaranteed to return a new, unique handle.
+sourcekitd_request_handle_t create_request_handle(void) {
```

Comment:
Can this function be `static`?

Reply:
Could idea. I hadn’t thought about that. 👍